.github/workflows/k8s-content.yaml

```diff
@@ -0,0 +1,167 @@
+---
+name: Kubernetes content
+
+on:
+  push:
+    branches: [ 'master' ]
+
+env:
+  COSIGN_EXPERIMENTAL: 1
+  IMAGE_NAME: ghcr.io/ComplianceAsCode/k8scontent
+
+jobs:
+  container:
+    runs-on: ubuntu-latest
+
+    permissions:
+      packages: write
+
+    outputs:
+      image-digest: ${{ steps.container_info.outputs.image-digest }}
+      image-tags: ${{ steps.container_info.outputs.image-tags }}
+
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Login to ghcr.io
+        uses: docker/login-action@v1.14.1
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      # TODO: Rename ocp4_content to k8s_content and make it more general
+      - name: Build container image
+        run: |
+          revision="$(git rev-parse "${GITHUB_REF_NAME:-HEAD}")"
+          docker build \
+            -f ./Dockerfiles/ocp4_content \
+            -t "${IMAGE_NAME}:latest" \
+            -t "${IMAGE_NAME}:${GITHUB_REF_NAME}" \
+            -t "${IMAGE_NAME}:${revision}" \
+            --label "org.opencontainers.image.source=https://github.com/ComplianceAsCode/content" \
+            --label "org.opencontainers.image.created=$(date --iso-8601=seconds)" \
+            --label "org.opencontainers.image.title=k8scontent" \
+            --label "org.opencontainers.image.revision=${revision}" \
+            --label "org.opencontainers.image.version=${GITHUB_REF_NAME}" \
+            --label "org.opencontainers.image.licenses=BSD" \
+            .
+
+      - name: Publish Container images
+        run: docker push "${IMAGE_NAME}" --all-tags
+
+      - name: Get container info
+        id: container_info
+        run: |
+          image_digest="$(docker inspect "${IMAGE_NAME}:latest" --format '{{ index .RepoDigests 0 }}' | awk -F '@' '{ print $2 }')"
+          image_tags="latest,${GITHUB_REF_NAME},$(git rev-parse "${GITHUB_REF_NAME:-HEAD}")"
+          echo "::set-output name=image-digest::${image_digest}"
+          echo "::set-output name=image-tags::${image_tags}"
+
+  sign:
+    runs-on: ubuntu-latest
+    needs: [container]
+
+    permissions:
+      packages: write
+      id-token: write
+
+    env:
+      IMAGE_DIGEST: ${{ needs.container.outputs.image-digest }}
+
+    steps:
+      - name: Install cosign
+        uses: sigstore/cosign-installer@v2.2.1
+        with:
+          cosign-release: v1.7.2
```

Comment:
Should we extract this into a variable so that we use the same version of cosign for all actions?

Reply:
I'd actually be up for removing the `with` statement and using whatever cosign release is recommended by the action. what do you think?